components/prism-python.js

```diff
@@ -14,3 +14,10 @@ Prism.languages.python= {
 	'operator' : /[-+]|<=?|>=?|!|={1,2}|&{1,2}|\|?\||\?|\*|\/|@|~|\^|%|\b(or|and|not)\b/,
 	'punctuation' : /[{}[\];(),.:]/
 };
+
+Prism.languages.insertBefore('python', 'keyword', {
```

Comment:
I don't see the need for the `insertBefore()` here. The `class-name` pattern could go directly into before the `keyword` key in the object above.